cmd/bblfsh/client.go

```diff
@@ -24,12 +25,45 @@ type clientCmd struct {
 	ImageRef    string `long:"image" value-name:"image-ref" description:"image reference to use (e.g. docker://bblfsh/python-driver:latest)"`
 	Language    string `long:"language" description:"language of the input" default:""`
 	Encoding    string `long:"encoding" description:"encoding used in the source file" default:"UTF8"`
+	CpuProfile  string `short:"" long:"cpuprofile" description:"path to file where Cpu Profile will be stored" default:""`
+	MemProfile  string `short:"" long:"memprofile" description:"path to file where Memory Profile will be stored" default:""`
 	Args        struct {
 		File string `positional-arg-name:"file" required:"true"`
 	} `positional-args:"yes"`
 }
 
+func (c *clientCmd) StartCpuProfilingMaybe() {
+	if c.CpuProfile != "" {
+		f, err := os.Create(c.CpuProfile)
+		if err != nil {
+			logrus.Errorf("Failed to create a CpuProfile file at %s, err:%s", c.CpuProfile, err)
+			os.Exit(1)
+		}
+		logrus.Infof("Start CPU profiling, save to %s", c.CpuProfile)
+		pprof.StartCPUProfile(f)
+	}
+}
+func (c *clientCmd) StopCpuProfilingMaybe() {
```

Comment:
* Not sure the `Maybe` is appropiate here, since you're calling the `StopCPUProfile` unconditionally.
* `StopCPU` would be more consistent with called function name.